src/index.js

```diff
@@ -1 +1,18 @@
 import './style.css';
+import loadData from './refreshData.js';
+import submitData from './submitData.js';
+
+const refresh = document.querySelector('.refresh');
+const formCnt = document.querySelector('.inputs');
+
+// Load stored data
+loadData();
```

Comment:
- [ ] **[OPTIONAL]:** I suggest that you don't run any global code, this can originate in some cases unexpected side effects! Instead, it would be great if you wrap this call to `loadData()` inside a `window.addEventListener('load')` listener, this way, you make sure that when the entire content loads, this event gets fired, and data gets loaded! 💯 🤓